kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/jdbc/JDBCMetadataStoreConf.scala

```diff
@@ -55,6 +55,7 @@ object JDBCMetadataStoreConf {
         "`org.apache.derby.jdbc.AutoloadedDriver`.</li>" +
         " <li>SQLITE: SQLite3, JDBC driver `org.sqlite.JDBC`.</li>" +
         " <li>MYSQL: MySQL, JDBC driver `com.mysql.cj.jdbc.Driver` " +
+        " <li>POSTGRESQL: PostgreSQL, JDBC driver `org.postgresql.Driver` " +
```

Comment:
looks like it should be swapped with the next line?

Reply:
Nice catch! Fixed it